### PR TITLE
Fixed sample product deploying issue once it delete

### PR DESCRIPTION
**Pull request by HiranyaKavishani**

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/SampleAPI/SampleAPIProduct.jsx [hunk 1]
```diff
@@ -67,45 +68,94 @@ const useStyles = makeStyles(theme => ({
...
+        const productRestApi = new APIProduct();
+
+        const productPromise = productRestApi.create(sampleProductPayload);
+        productPromise
+            .then((prod) => {
+                setStep(4);
+                setProductPath(`/api-products/${prod.body.id}/overview`);
+                Alert.info(intl.formatMessage({
+                    id: 'Apis.Listing.SampleAPI.SampleAPIProduct.successful',
+                    defaultMessage: 'CalculatorAPIProduct published successfully',
```
Comment (harsha89) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/SampleAPI/SampleAPIProduct.jsx]: How about saying Sample API Product published successfully.

Comment (HiranyaKavishani) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/SampleAPI/SampleAPIProduct.jsx]: Modified it to 'Sample CalculatorAPIProduct' as  we have made the sample api message as  'Sample Pizzashack API'

Comment (tmkasun) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/SampleAPI/SampleAPIProduct.jsx]: Yes I think it's better to say 

```suggestion
                    defaultMessage: 'Sample API Product published successfully',
```

Comment (HiranyaKavishani) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/SampleAPI/SampleAPIProduct.jsx]: Don't we need to add the name of the APIProduct?

**Pull request merged**